Update readme to clarify position on third-party PRs. (#13019)
google/site-kit-wp: README.md

## Contributing

> We welcome bug reports and feature requests, though it is preferable to [send both to the Site Kit support forums on WordPress.org](https://wordpress.org/support/topic/read-first-welcome-to-the-site-kit-support-forum-5/).

Reviewer: ```suggestion
We welcome feature requests and bug reports through [our official support forum on WordPress.org](https://wordpress.org/support/topic/read-first-welcome-to-the-site-kit-support-forum-5/) where we actively monitor and reply to all topics.
```

> 

Reviewer: ```suggestion

For reports of a security-related nature, please refer to [our security policy](https://github.com/google/site-kit-wp/security/policy).

```

> While we'd like to accept third-party code contributions to Site Kit, we are a large team with an established, pre-planned, sprint-based workflow that includes non-GitHub tools like ZenHub. We manages tasks using ZenHub, plan sprints in advance, and use tools which are not visible to external contibutors.

Reviewer: ```suggestion
While we'd like to accept third-party code contributions, we are a large team with an established, pre-planned, regimented workflow that includes tools that are not available to external contributors.
```

We also have guidelines around when an issue is ready to be worked on, including having complete Acceptence Criteria and Implementation Briefs signed off by code owners. All contributors must also [sign the Google Contributor License Agreement](https://cla.developers.google.com/about/google-individual).

For these reasons we currently discourage third-party contributions to Site Kit because the review team is not well-equipt to handle external contributions.

If you still want to make a contribution, you can read the [Contributor Handbook](https://github.com/google/site-kit-wp/wiki) to get started, or the [Engineering set up quickstart](https://github.com/google/site-kit-wp/wiki/Engineering#set-up-site-kit-project) to set up Site Kit locally.

> But please understand we may not be able to prioritize or review your contribution, so at this time we encourage you to focus on bug reports and feature requests. 🙂

Reviewer: I think we can remove these paragraphs, otherwise it's sending an odd mixed message.
```suggestion
```

## Requirements

-   WordPress >= 5.2

> -   PHP >= 7.4

Reviewer: This change seems unnecessary 🤔

Author: Oh, indeed, must've just been an editor glitch. I'll revert it 👍🏻
